### PR TITLE
Prettier config

### PR DESCRIPTION
**Pull request by mikewheaton**
Adds a `.prettierrc` config file, with settings copied from [Fast5](https://github.com/get-convex/fast5/blob/main/.prettierrc), to ensure that files are formatted consistently for this project regardless of local settings. Formats all of the files so we have a consistent starting point and avoid unnecessary diffs in future PRs.

### CODE REVIEW

**Review by thomasballinger — APPROVED** [review]

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
```
[the remark this answers is not in the document]
Comment (mikewheaton) [reply .prettierrc:2]: I'm good with that, and it will cut down on the diff here too. Trying it out now.

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (mikewheaton) [inline .prettierrc:1]: This was a little odd. If I remove `.prettierrc` entirely, when I save in VSCode it seems to be picking up a config file elsewhere on my system. That gives an inconsistent result from running `npx prettier --write .`. By including this empty config file, both format-on-save and explicitly running the Prettier command have the same output.

Comment (thomasballinger) [reply .prettierrc:1]: makes sense, we need something here

**Review by thomasballinger — APPROVED** [review]

File: .prettierrc (hunk 2 above)
Comment (mikewheaton) [reply .prettierrc:1]: I guess my mental model was of per-setting overrides, but it does make sense that the entire configuration is one object.

**Pull request merged**